third_party/flower/BUILD.gn

```diff
@@ -0,0 +1,22 @@
+import("//third_party/grpc/grpc_library.gni")
+
+static_library("flower") {
+  include_dirs = [ "src/cc/flwr/include" ]
+  sources = [
+    "src/brave/flwr/serde.cc",
+    "src/brave/flwr/serde.h",
+  ]
+  deps = [
+    "//brave/third_party/flower/src/proto/flwr/proto:transport_grpc_proto",
+    "//components/grpc_support",
+    "//net",
+    "//third_party/grpc:grpc++",
+    "//third_party/protobuf:protobuf_lite",
+  ]
+
+  configs -= [ "//build/config/compiler:chromium_code" ]
+  configs += [
+    "//build/config/compiler:no_chromium_code",
+    "//build/config/compiler:exceptions",
```

Comment:
Hmm, is it safe? What if the code throw an extension in the browser process?
cc @goodov

Reply:
@LorenzoMinto are there some compilation errors? why did you add this config?

Reply:
removed them. These were residues from a previous iteration